Registration/README.md

```diff
@@ -41,6 +41,23 @@ Set-AzsRegistration -PrivilegedEndpoint "<Computer Name>-ERCS01" -BillingModel P
 Set-AzsRegistration -PrivilegedEndpoint "<Computer Name>-ERCS01" -BillingModel PayAsYouUse -MarketplaceSyndicationEnabled:$false
 ```
 
+## Re-Register in a connected environment
+In a connected environment, to re-register with Azure using the existing registration details, set the correct Azure PowerShell context using the subscription used during the existing registration. 
```

Comment:
We can deduce the right subscription to use from the existing registration resource, which should include the Azure subscription ID. In that case, we just need to ensure the user has authenticated using an account that has access to registered subscription. So perhaps, we could instead have a note here asking them to authenticate to "any" of their Azure subscriptions, and the reregister flow will attempt to select the right subscription for them.

Then, in the code below, when we retrieve the existing registration resource, we can turn-around and call Select-AzureRmSubcription for the azure environment, and after that it would be as though they selected the correct subscription all-along. We can even trace something like "Existing registration found against subscription <x>, attempting to select this subscription now" and then displaying error if that fails like "Existing registration was against azure subscription with id '<x>', please use an account with access to that subscription, or if that subscription no-longer exists, remove the existing registration, and register using a subscription to which you do have access"

Reply:
+1